opencti-platform/opencti-front/tests_e2e/settings/parameters.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test } from '@playwright/test';
+import { expect } from '../fixtures/baseFixtures';
+import { LeftBarPage } from '../model/menu/leftBar.pageModel';
+
+test('Check EE activation', async ({ page }) => {
+  const leftBarPage = new LeftBarPage(page);
+
+  await page.goto('/');
+  await page.getByTestId('ChevronRightIcon').click();
+  await leftBarPage.clickOnMenu('Settings', 'Parameters');
+
+  await page.getByRole('button', { name: 'Enable Enterprise Edition' }).click();
```

Comment:
Maybe we could create a page model for EE stuff as it is present on several pages?

Reply:
In the future we may need to do that yes. But that check is specifically for the setting part.